app/models/reservation.rb

```diff
@@ -0,0 +1,2 @@
+class Reservation < ApplicationRecord
```

Comment:
Consider providing an association between Reservation and Ticket (and later between other models)

Reply:
This is what I wanted to do in the next commit :)